### DISCUSSION TITLE
Quick terminal discoverability on MacOS

### DISCUSSION THREAD

**Discussion by cwindolf** [ghostty-org/ghostty#4098] [Feature Requests, Ideas]
Hi, new to Ghostty, thanks for the very nice app. I got confused while setting up the quick terminal (until I found https://github.com/ghostty-org/ghostty/discussions/3610) and thought I would mention two things that could reduce friction for future users.
- The shortcut (configured as in https://ghostty.org/docs/config/keybind/reference#toggle_quick_terminal) does not show up in Ghostty's View menu. Not sure if it's possible for the user-configured shortcut to appear there, but it would help me reference it when I forget.
- After giving Ghostty the required Accessibility permission, the app needs to be restarted before the `global:` shortcut works while other apps are active.

**Comment by canTuncPekkan** [comment]
<img width="206" alt="Screenshot 2024-12-30 at 9 07 10 PM" src="https://github.com/user-attachments/assets/c0c136c2-c207-489f-b957-8278e05a4c31" />


what do you mean it is there it is second from the bottom, my short cut is cmd + .

**Comment by cwindolf** [comment]
Thanks for the reply! Huh, doesn't show up on my end.

<img width="197" alt="Screenshot 2024-12-30 at 1 09 29 PM" src="https://github.com/user-attachments/assets/8c488630-4989-4056-a10e-41f5e71c5137" />

**Reply by canTuncPekkan** [reply]
That is wired I changed my keybind multiple times to check when it updated on the drop down menu, and it was instant.

I don't think that it is a problem of length as `shift+ctrl+opt+cmd+.` works fine on my computer

what is your short cut maybe ghostty has problems showing some key binds

**Comment by cwindolf** [comment]
Yeah, my config has the line `keybind = global:alt+space=toggle_quick_terminal`. Not requesting that this be configured by default, I was just wondering why my shortcut doesn't show up next to the option in the View menu like @canTuncPekkan 's does.

And, @canTuncPekkan's speculation about certain keybindings being the problem seems to be true! I think having a space in the binding was causing the problem. If I change to alt+., it shows up fine! Thanks for your help -- guess that's what I was running into

**Reply by acsetter** [reply]
Interesting, did the binding still work with the space there?

**Reply by cwindolf** [reply]
Yeah, the binding does work with the space.

**Reply by canTuncPekkan** [reply]
do you think this a bug in macOS or ghostty

**Reply by acsetter** [reply]
Yupp, I'm seeing the same thing. I wonder if this is just an Apple thing where the space-bar does not have an icon, so the tip does not appear in the menu?

**Reply by cwindolf** [reply]
I don't think it's a macOS thing. I just used System Settings to add ctrl+space as a shortcut to a menu item (Window > Zoom) in Firefox as a test, and it shows up as:
<img width="748" alt="Screenshot 2024-12-31 at 10 53 23 AM" src="https://github.com/user-attachments/assets/d3551eb8-0dd7-42a5-ac06-96d1feeb37f5" />
